config.go

```diff
@@ -199,6 +201,8 @@ type Config struct {
 	TaprootAssetsMode string         `long:"taproot-assets-mode" description:"The mode to run taproot assets in, either 'integrated' (default), 'remote' or 'disable'. 'integrated' means tapd is started alongside the UI and everything is stored in tap's main data directory, configure everything by using the --taproot-assets.* flags. 'remote' means the UI connects to an existing tapd node and acts as a proxy for gRPC calls to it. 'disable' means that LiT is started without tapd" choice:"integrated" choice:"disable"`
 	TaprootAssets     *tapcfg.Config `group:"Integrated taproot assets options (use when taproot-assets=integrated)" namespace:"taproot-assets"`
 
+	AccountsMode string `long:"accounts-mode" description:"The mode to run the accounts service in, either 'enable' (default), or 'disable'. 'enable' means that LiT is started with the integrated the accounts service enabled and running. 'disable' means that LiT is started with the accounts service disabled, and all requests to the accounts service will be rejected." choice:"enable" choice:"disable"`
```

Comment:
could also just be a `disable` boolean instead?

Reply:
Good idea! Thanks, updated with a boolean instead :)!